tests/pools/thread/fixed.test.js

```diff
@@ -80,9 +90,10 @@ describe('Fixed thread pool test suite', () => {
     } catch (e) {
       inError = e
     }
-    expect(inError).toBeTruthy()
-    expect(inError instanceof Error).toBeTruthy()
-    expect(inError.message).toBeTruthy()
+    expect(inError).toBeDefined()
+    expect(inError).toBeInstanceOf(Error)
+    expect(inError.message).toBeDefined()
+    expect(typeof inError.message === 'string').toEqual(true)
```

Comment:
Why not `.toBeTruthy()`? I think we can safely use it here

Reply:
Stricter returned value check: typeof return a boolean, never null, '', etc.

Reply:
But a `typeof ... ===` check can only return true or false

Reply:
> But a `typeof ... ===` check can only return true or false

So tests now strictly check for a boolean 'true' value now ;)